### ISSUE TITLE
Unwanted attributes listed through RPC

### ISSUE THREAD

**Issue by janpavelkao2** [O2-Czech-Republic/proxima-platform#193]
Listing attributes throught RPC returns attributes that shouldn't be returned. E.g. listing "device." results in "device." and "wifiDevice." being listed.

**Comment by je-ik** [comment]
Looks related to `RandomHBaseReader.java:117`:
```java
      if (stroff != null) {
        scan.setFilter(
            new ColumnPaginationFilter(
                limit, (stroff.getOffset() + '\00').getBytes(StandardCharsets.UTF_8)));
      }
```
The filter should probably be constructed as combination of the prefix filter.

**Comment by je-ik** [comment]
Closed via datadrivencz#492
